src/wmtk/operations/polygon_mesh/MakeEdge.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include <wmtk/operations/TupleOperation.hpp>
+#include "PolygonMeshOperation.hpp"
+
+namespace wmtk::operations {
+namespace polygon_mesh {
+class MakeEdge;
+}
+
+template <>
+struct OperationSettings<polygon_mesh::MakeEdge>
+{
+    // TODO Determine if operation settings are necessary
+};
+
+namespace polygon_mesh {
+
+/**
+ * @class Atomic operation for adding a bubble component with a single edge.
+ */
+class MakeEdge : public PolygonMeshOperation, private TupleOperation
+{
+public:
+    // constructor for default factory pattern construction
+    MakeEdge(Mesh& m, const OperationSettings<MakeEdge>& settings);
+    MakeEdge(PolygonMesh& m, const OperationSettings<MakeEdge>& settings);
+
+    std::string name() const override;
+
+    /**
+     * @brief Return tuple corresponding to the created edge
```

Comment:
which vertex does it refer to?

Reply:
I'm not sure I follow. It makes a new connected component with a new face, a new edge, and two new vertices.

Reply:
I see. In this case, the vertices of the bubble are perfectly symmetric, so I can note that it's just an arbitrary vertex. 

Of more concern I suppose is whether the vertex is the tip or base of the halfedge of the tuple. Generally, I have been using the base of the halfedge as the default so that `next(h)` is the same as `switch_edge(switch_vertex(h))`. I can note this.